src/index.js

```diff
@@ -56,10 +56,12 @@ const validateData = (data) => {
   }
 
   if (!data.games.every(game =>
-    (game.home &&
-     game.away &&
-     Number.isInteger(game.home) &&
-     Number.isInteger(game.away))
+    (
+      {}.hasOwnProperty.call(game, 'home') &&
```

Comment:
why not `game.hasOwnProperty('home')`? That looks a lot more natural than creating an object instance from scratch